### DISCUSSION TITLE
Map of Ukraine on Speed Insights

### DISCUSSION THREAD

**Discussion by hyzyla** [vercel/community#5140] [Bug Report]
Dear Vercel Team,

I am writing to draw your attention to a matter concerning the map displayed on the Speed Insights page of your dashboard.

I have noticed that the current map representation does not include Crimea as part of Ukraine. This depiction is inconsistent with the United Nations’ recognition of Ukraine’s borders, which includes Crimea.

I kindly request that you consider updating the map data used on the Speed Insights page to reflect the internationally recognized borders of Ukraine, including Crimea. Such an update would ensure accuracy and compliance with international understanding and agreements

![image](https://github.com/vercel/community/assets/7952949/4dd7cc6c-41db-4fe6-8b4c-f1e2979f1a88)

**Comment by leerob** [comment]
Updated, thank you 👍
